src/python/pants/backend/python/rules/python_fmt.py

```diff
@@ -22,6 +26,7 @@
 from pants.engine.rules import UnionRule, optionable_rule, rule
 from pants.engine.selectors import Get
 from pants.rules.core.fmt import FmtResult, FmtTarget
```

Comment:
I think FmtTarget should be renamed TargetWithSources or something like that. Now that we're using it outside the Fmt context.

Reply:
Addressed in new commit in this PR.